app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseScreen.kt

```diff
@@ -28,6 +33,8 @@ abstract class BaseScreen<K : ScreenKey, B : ViewBinding, M : Parcelable, E, F,
     private const val KEY_MODEL = "org.simple.clinic.navigation.v2.fragments.BaseScreen.KEY_MODEL"
   }
 
+  private lateinit var _viewModel: MobiusLoopViewModel<M, E, F, V>
```

Comment:
Is there a reason we are prefixing this with `_`?

Reply:
My initial plan was to use this as a private reference and then expose the `viewModel` using a getter similar to binding. So I left it in case we decided to do that in future. But we can remove this if you think we don't want to expose the view model.

Reply:
I renamed it, we can make this change when needed